### ISSUE TITLE
Unknow custom element: <fa>

### ISSUE THREAD

**Issue by Sebastiangperez** [nuxt-community/fontawesome-module#21]
hi guys, i got this error and i did everything that you said in the docs 

Im using the latest version of nuxt.js

Unknown custom element: <fa> - did you register the component correctly? For recursive components, make sure to provide the "name" option.

found in

---> <Pages/index.vue> at pages/index.vue
       <Nuxt>
         <Layouts/default.vue> at layouts/default.vue
           <Root>

Any tougths ?

**Comment by josh-unwin** [comment]
I'm getting the same thing. Finding the docs quite confusing.

**Comment by Sebastiangperez** [comment]
@josh-unwin , yes, is not very helpfull, they need to change a little bit the documentation, i have to install another package for node
`"@fortawesome/fontawesome-free": "^5.15.1",` 

and i have this in the nuxt

`'@nuxtjs/fontawesome',`

and this in nuxt.config.js 

```
fontawesome: {
    component: 'fa',
    icons: {
      solid: true,
      brands:true

    }
  },

```
Maybe this will help you. and i uses like this :

`<fa icon="film" />`

**Comment by Sebastiangperez** [comment]
@josh-unwin did it work ??

**Comment by josh-unwin** [comment]
Hey thanks @Sebastiangperez, I only just went back to this project, yep that's working now! Thanks!

**Comment by pimlie** [comment]
@Sebastiangperez _They_ includes you, you are part of the nuxt-community too! ;)

Could you please have a look at submitting a PR with changes to make the docs more understandable?

**Comment by JoseFortunato** [comment]
Hello,

I have similar issue. I have <fa :icon="['fas', 'adjust']" /> working perfectly in pages, but when a try to use in components and layouts, i have de same message: `Unknown custom element: <fa> - did you register the component correctly?`

I tried the suggestion of @Sebastiangperez, but didn't work to me.

**Comment by schneefux** [comment]
See the `component` option:
> component
> Default: FontAwesomeIcon

Installing another package is not necessary. You need to use `<FontAwesomeIcon>` instead of `<fa>`. `<fa>` is only available if you set `component: 'fa'` in the options. The readme examples mix both styles.
